common/src/main/java/feast/common/function/Feature.java

```diff
@@ -14,25 +14,28 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package feast.serving.util;
+package feast.common.function;
```

Comment:
Why do we have to nest this inside of `function`? I thought we wanted to have the functionality be domain specific. How about `models`?